src/apps/viewer/index.html

```diff
@@ -35,6 +35,7 @@
             }
         </style>
         <link rel="stylesheet" type="text/css" href="molstar.css" />
+        <link rel="manifest" href="./manifest.webmanifest">
```

Comment:
Instead of having this in the default all, would it make sense to instead create a `sw.html` which supports this instead to make it optional?